lib/kc_elixir_web/templates/page/index.html.eex

```diff
@@ -15,6 +15,27 @@
       <li>
         <strong>Topic</strong>: <%= meetup.topic %>
       </li>
+      <li>
+        <strong>Speaker Info</strong>:
```

Comment:
I think we could probably collapse all of this `Speaker Info` into one short line. It'd also be nice to have a user avatar to show as well.

Something along the lines of this I think might be concise and usable, though with good styling and conditionals to gracefully degrade when we have limited speaker metadata

+ **Speaker**: `avatar` `name` [github icon] [twitter icon]

Where `avatar` and `name` can link to the speaker's URL, and various social media icons link to their respective URLs.